vendor/engines/nucore_kfs/lib/nucore_kfs/journal_closer.rb

```diff
@@ -0,0 +1,8 @@
+module NucoreKfs
+  class JournalCloser < ::Journals::Closer
```

Comment:
This feels a little future-proofy - I know, I'm the one that sent you down this path :) but looking at this implementation, I wonder how likely you are going to be to need to make more changes to the journal closing behavior.  

I think that overriding `after_success` could be achieved more simply from your [engine initializer](https://github.com/SquaredLabs/nucore-uconn/blob/master/vendor/engines/nucore_kfs/lib/nucore_kfs/engine.rb) like:
```
config.to_prepare do
  ...
  Journals::Closer.send(:include, NucoreKfs::JournalsCloserExtension)
  ...
end
```
Then you'd change the class here to:
```suggestion
  class JournalsCloserExtension
```
... and we don't need to mess with `cattr_accessor(:journal_closer) { Journals::Closer }` in the controller.

If you've got more changes in the works for this part of the code, then adding the `NucoreKfs::Journals::Closer` class now would be a good move.  If not, I would stick with the simpler approach.  What do you think?

Reply:
We don't have more changes in this part of the code. To be honest, I don't think I have enough rails experience to determine which of these approaches is better long term, so I'm happy to defer to your judgement. We will go with the simpler approach.

I'll work on making this change and let you know when it is ready for another review.

Reply:
To do this, we would also need to change the original `Journals::Closer` class to have the `after_success` function inside of `Overridable` like this:

```
  module Overridable
    def after_success
      # no-op by default. Can be hooked into by engines to provide school-specific behavior
    end
  end
  ```
  
At least, that seems to be necessary based on my testing. Does that look correct to you?